src/ToDoView.test.js

```diff
@@ -1,13 +1,14 @@
 import ToDoView from './ToDoView';
 import userEvent from '@testing-library/user-event';
-import { render, screen, waitForElementToBeRemoved } from '@testing-library/react';
+import { render, screen, waitForElementToBeRemoved, within } from '@testing-library/react';
 import axios from 'axios';
 import { act } from 'react-dom/test-utils';
 
-const addButton = () => screen.getByRole('button', '+ Add To Do');
-const saveButton = () => screen.getByText('Add To Do');
-const cancelButton = () => screen.getByText('Cancel');
+const addButton = () => screen.getByRole('button', { name: '+ Add To Do' });
+const saveButton = () => screen.getByRole('button', { name: 'Add To Do' });
+const cancelButton = () => screen.getByRole('button', { name: 'Cancel' });
 const toDoTextField = () => screen.getByLabelText('Enter To Do here');
+const mainToDoList = () => screen.getByRole('list');
 
 jest.mock('axios');
```

Comment:
this becomes jest.mock('./ToDoService') such that you have no references to axios in your component